packages/grafana-ui/src/components/Select/Select.story.tsx

```diff
@@ -276,14 +285,18 @@ export const CustomValueCreation = () => {
         onChange={(v) => {
           setValue(v);
         }}
-        allowCustomValue
+        allowCustomValue={args.allowCustomValue}
         onCreateOption={(v) => {
           const customValue: SelectableValue<string> = { value: kebabCase(v), label: v };
           setCustomOptions([...customOptions, customValue]);
           setValue(customValue);
         }}
-        {...getDynamicProps()}
+        prefix={getPrefix(args.icon)}
+        {...args}
       />
     </>
   );
 };
+CustomValueCreation.args = {
+  allowCustomValue: false,
```

Comment:
I think the whole point of this story is to show the custom creation option so maybe this should be
```suggestion
  allowCustomValue: true,
```

Reply:
okidoki 👍